### PR TITLE
chore: resolve issues for focus(fix #3, #4)

### LINKED ISSUE #3: bug: focus is different when clicked and when selected with keyboard.
## Current Behavior

* The focus behavior is different when clicked and when selected with keyboard.
  * After clicking to select, the selectbox does not have focus.

keyboard
* keyboard
mouse
* mouse

## Expected Behavior
* After selecting, there should always be focus.

### LINKED ISSUE #4: enhancement: focus API
## Current Behavior
* none

## Expected Behavior

```js
const selectBox = new SelectBox({...});

selectBox.focus();
```

### PR DESCRIPTION
**Pull request by jajugoguma**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
* add focus api(fix #4)
* fix to maintain focus with mouse click(fix #3)

**as-is**

**to-be**

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by jung-han — APPROVED** [review]
완료입니다~

File: test/selectbox.spec.js [hunk 1]
```diff
@@ -58,6 +58,15 @@ describe('SelectBox', () => {
     expect(dropdown.el).toHaveClass(cls.HIDDEN);
   });
 
+  it('should focus input when the focus api call', () => {
+    const { input } = selectBox;
```
Comment (lja1018) [inline test/selectbox.spec.js:62]: `const { el } = selectBox.input;`으로 해도 될 것 같아요.

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

**Review by jungeun-cho — APPROVED** [review]
이견없습니다.

**Pull request merged**